core/src/font.rs

```diff
@@ -249,6 +249,38 @@ impl FontFace {
 
         Twips::ZERO
     }
+
+    pub fn is_valid(bytes: &[u8]) -> bool {
```

Comment:
I don't think we need to add these as static methods, instead of calling them directly.

Reply:
Wanted to avoid to expose ttf_parser function to the web but is better to avoid multiple parsing and loading.